x-pack/plugins/apm/public/components/shared/apm_header_action_menu/alerting_popover_flyout.tsx

```diff
@@ -86,16 +82,26 @@ export function AlertingPopoverAndFlyout({
         ...(canSaveAlerts
           ? [
               {
-                name: transactionDurationLabel,
-                panel: CREATE_TRANSACTION_DURATION_ALERT_PANEL_ID,
-              },
-              {
-                name: transactionErrorRateLabel,
-                panel: CREATE_TRANSACTION_ERROR_RATE_ALERT_PANEL_ID,
+                name: createThresholdAlertLabel,
+                panel: CREATE_THRESHOLD_PANEL_ID,
               },
+              ...(canReadAnomalies
```

Comment:
Can't these define just `name` and `panel` and then have separate entries for the panels with `name` / `onClick` to keep them consistent with the threshold panel?